app/components/UserProfile/SecondaryProfile.tsx

```diff
@@ -17,17 +19,36 @@ const SecondaryProfile: FC<SecondaryProfileProps> = ({
   avatarWidth = 40,
   className = "",
   flexOrder,
+  shouldOpenModal,
 }) => {
+  const [open, setOpen] = useState(false);
+
+  const anchorRef = useRef(null);
```

Comment:
Properly type useRef (anchorRef as HTMLDivElement | null for type safety).

```suggestion
  const [open, setOpen] = useState<boolean>(false);
  const anchorRef = useRef<HTMLDivElement | null>(null);
```

Reply:
done